core/src/main/scala/io/finch/Outputs.scala

```diff
@@ -4,47 +4,66 @@ import com.twitter.finagle.http.Status
 
 trait Outputs {
 
-  // See https://gist.github.com/vkostyukov/32c84c0c01789425c29a to understand how this list is assembled.
+  // See https://gist.github.com/vkostyukov/32c84c0c01789425c29a to understand how this list
+  // is assembled.
 
   // 2xx
-  def Ok[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Ok)                                               // 200
-  def Created[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Created)                                     // 201
-  def Accepted[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Accepted)                                   // 202
-  def NoContent[A](a: A): Output.Payload[A] = Output.Payload(a, Status.NoContent)                                 // 204
+  def Ok[A](a: A): Output[A] = Output.payload(a, Status.Ok)                                   // 200
+  def Created[A](a: A): Output[A] = Output.payload(a, Status.Created)                         // 201
+  val Accepted: Output[Unit] = Output.unit(Status.Accepted)                                   // 202
```

Comment:
Seeing as Accepted "SHOULD include an indication of the request's current status and either a pointer to a status monitor or some estimate of when the user can expect the request to be fulfilled" would it not be better to make this `Output.payload` to encourage users to include a payload?